src/coreclr/vm/ceeload.inl

```diff
@@ -35,6 +35,7 @@ void LookupMap<TYPE>::SetValueAt(PTR_TADDR pValue, TYPE value, TADDR flags)
 
     value = dac_cast<TYPE>((dac_cast<TADDR>(value) | flags));
 
+    // REVIEW: why this is not VolatileStore? What guarantees that we do not have concurrent readers?
```

Comment:
I think `LookupMap<TYPE>` _can_ be read concurrently. I will look at this separately. Maybe it is ok. 
It would be a separate issue anyways